### DISCUSSION TITLE
saving credentials from multiple access points easily possible?

### DISCUSSION THREAD

**Discussion by ghmpi** [Hieromon/AutoConnect#536] [How can I?]
Hello,

I would like to be able to save the credentials of at least two different access points with AutoConnect. I don't believe this is possible right now unless I missed something obvious.

Would this be something that would be easy to add? Either as a code add on I can do (easily!) or an actual feature with AutoConnect?

This project has been working pretty well for me BTW!

Thank you!

Regards,
-Moses

**Comment by Hieromon** [comment]
I'm not sure I understand your question. AutoConnect can manage credentials for two or more different access points as different SSID. This is the default behavior, but is your intent elsewhere?

**Reply by ghmpi** [reply]
Hieromon,

I was under the impression, maybe incorrectly, that AutoConnect is not able to store credentials for more then one access point/SSID. I'll test my setup again and see what I missed.

Thank you for the quick reply.

Regards,
-Moses

**Comment by ghmpi** [comment]
I just came back to this. I see now that if I connect to more then one wifi.. it will remember them and add them to the "Open SSIDs" menu.

I believe I was using a really old version before, not sure if anything was different.

Now I know that I can have at least 2 different SSIDs saved with version 1.4.0.

Thank you!

Regards,
-Moses
